packages/babel-traverse/scripts/generators/validators.js

```diff
@@ -9,6 +9,7 @@ export default function generateValidators() {
  */
 import * as t from "@babel/types";
 import NodePath from "../index";
+import { VirtualTypeAliases } from "./virtual-types";
```

Comment:
Nit: can use type only imports since the generated `validators` contains only typings.